docs/content/latest/integrations/smart-driver.md

```diff
@@ -0,0 +1,174 @@
+  ds.setTopologyKeys("cloud1.region1.zone1,cloud1.region2.zone2");
+  // Provide more end points to prevent first connection failure 
+  // if an initial contact point is not available 
+  ds.setAdditionalEndpoints("127.0.0.2:5433,127.0.0.3:5433");
+  
+  Connection conn = ds.getConnection();
+  ```
+
+- Configure `YBClusterAwareDataSource` with a pooling solution such as Hikari and then use it to create a connection, as per the following example:
+
+  ```java
+  Properties poolProperties = new Properties();
+  poolProperties.setProperty("dataSourceClassName", "com.yugabyte.ysql.YBClusterAwareDataSource");
+  poolProperties.setProperty("maximumPoolSize", 10);
+  poolProperties.setProperty("dataSource.serverName", "127.0.0.1");
+  poolProperties.setProperty("dataSource.portNumber", "5433");
+  poolProperties.setProperty("dataSource.databaseName", "yugabyte");
+  poolProperties.setProperty("dataSource.user", "yugabyte");
+  poolProperties.setProperty("dataSource.password", "yugabyte");
+  poolProperties.setProperty("dataSource.loadBalance", "true");
+  // Provide additional end points
+  String additionalEndpoints = "127.0.0.2:5433,127.0.0.3:5433,127.0.0.4:5433,127.0.0.5:5433";
+  poolProperties.setProperty("dataSource.additionalEndpoints", additionalEndpoints);
+  // Load balance between specific geo-locations using topology keys
+  String geoLocations = "cloud1.region1.zone1,cloud1.region2.zone2";
+  poolProperties.setProperty("dataSource.topologyKeys", geoLocations);
+  
+  poolProperties.setProperty("poolName", name);
+  
+  HikariConfig config = new HikariConfig(poolProperties);
+  config.validate();
+  HikariDataSource ds = new HikariDataSource(config);
+  
+  Connection conn = ds.getConnection();
+  ```
+
+## Examples
+
+To access a sample application that uses Yugabyte Smart Driver, visit [YugabyteDB JDBC driver](https://github.com/yugabyte/jdbc-yugabytedb).
+
+To be able to use the samples, you need to complete the following steps: 
+
+- Install YugabyteDB by following instructions provided in [Quick Start Guide](https://docs.yugabyte.com/latest/quick-start/install/). 
+
+- Build the examples by running `mvn package`.
+
+- Run the `run.sh` script, as per the following guideline:
+
+  ```
+  ./run.sh [-v] [-i] -D -</path_to_yugabyte_installation/>
+  ```
+
+  In the precediding command, replace:
+
+  - *[-v] [-i]* &nbsp; with `-v` if you want to run the script in `VERBOSE` mode.
+
+  - *[-v] [-i]* &nbsp; with `-i` if you want to run the script in `INTERACTIVE` mode.
+
+  - *[-v] [-i]* &nbsp; with `-v` `-i` if you want to run the script in both `VERBOSE` and `INTERACTIVE` mode at the same time.
+
+  - *</path_to_yugabyte_installation/>* &nbsp; with the path to the directory where you installed YugabyteDB. 
+
+  <br>For more details, you can consult the help for the script.<br>
+
+  <br> The following is an example of a shell command that runs the script:
+
+  ```sh
+  ./run.sh -v -i -D ~/yugabyte-2.7.1.1/
```

Comment:
This version, 2.7.1.1 is old. And we cannot provide a version yet as it is not pub;ished yet. In the works. Will keep you posted about this.

Reply:
Okay

Reply:
2.7.1.1 is still there. We need to fix it and for that we need to know the exact version which we don't have right now. Will talk to @rao-vasireddy and get back

Reply:
We can replace the version with the user-defined version for now:
`./run.sh -v -i -D ~/yugabyte-<yugabytedb-version>/`

You need to replace _yugabytedb-version_ with the version of YugabyteDB you are using. (or something like that)

Then we'll edit the doc when the actual version is available.